templates/c/README.md

```diff
@@ -0,0 +1,27 @@
+# C Starter Project Template 
+
+This is a C / TIC-80 starter template.  To build it:
+
+```
+export WASI_SDK_PATH=
```

Comment:
Add a link to WASI above this maybe?  And an example here?

`/where/you/installed/WASI` ?

Thoughts?

Reply:
sounds good, I'll add it.